lib/src/iterable_extensions.dart

```diff
@@ -793,3 +826,66 @@ extension ComparatorExtension<T> on Comparator<T> {
         return result;
       };
 }
+
+/// Extensions on sets.
+extension SetExtension<T> on Set<T> {
+  /// Whether this set has the same unordered elements as [elements].
+  ///
+  /// Checks whether this set has the same elements as [elements],
+  /// but not necessarily in the same iteration order.
+  ///
+  /// Uses [equality] for determining if the values are the same.
+  /// If [equality] is omitted, default to the equality used by
+  /// this set.
+  /// _(Notice: This differs from [SetEquality],
+  /// which is symmetric in the arguments)_.
```

Comment:
I don't think I understand this statement, and it doesn't look like the `SetEquality` docs mention "symmetric". Can you expand this?